### PR TITLE
HPCC-28927 Add global/expert settings to container doc

### PR DESCRIPTION
**Pull request by g-pan**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

successful integration testing completed on : http://10.224.20.18/view/Docs-gp/job/DocBuild-01-GPRepo

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 1]
```diff
@@ -1319,6 +1319,83 @@ global:
         <para>You can specify the override flags multiple times. The priority
         will be given to the last (right-most) file specified.</para>
       </sect3>
+
+      <sect3 id="YGlobalExpertsets">
+        <title>Global/Expert Settings</title>
+
+        <para>The 'expert' section under 'global' of the values.yaml should be
+        used to define low-level, testing or developer settings. This section
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1327]: needs a serial comma: low-level, testing, or developer settings

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 2]
```diff
@@ -1319,6 +1319,83 @@ global:
...
+            <varlistentry>
+              <term>maxConnections</term>
+
+              <listitem>
+                <para>(unsigned) This is a DFU Server setting. If set, it will
+                limit the maximum number of parallel connections and partition
+                streams that will be active at any one time. By default a DFU
+                job will run as many active connection/streams as there are
+                partitions involved in the spray, limited to an absolute
+                maximum of 800. Setting maxConnections can be set to reduce
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1375]: Setting maxConnections can be set to reduce this concurrency.
s/b 
The maxConnections setting can be used to reduce this concurrency.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 3]
```diff
@@ -1319,6 +1319,83 @@ global:
...
+                degraded performance.</para>
+              </listitem>
+            </varlistentry>
+
+            <varlistentry>
+              <term>keepalive</term>
+
+              <listitem>
+                <para> (time: unsigned, interval: unsigned, probes: unsigned)
+                See keepalive example above. If set, these settings will
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1387]: See keepalive example above.
I cannot find the example.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 4]
```diff
@@ -1319,6 +1319,83 @@ global:
...
+                override the system default socket keepalive settings each
+                time the platform creates a socket. This may be useful in some
+                scenarios if the connections would otherwise be closed
+                prematurely by external factors (e.g. firewalls). An example
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1391]: s/b
(e.g., firewalls)

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
comments inline

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml (hunk 3 above)
Comment (jakesmith) [reply docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1387]: line 1339->1342

**Review by jakesmith — APPROVED** [review]
@g-pan - Aside from @JamesDeFabia 's minor comments, looks ok to me.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml (hunk 3 above)
Comment (JamesDeFabia) [reply docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1387]: Thanks

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV

**Comment by ghalliday** [comment]
@g-pan please squash

**Comment by g-pan** [comment]
squashed. should be good to merge now.

**Pull request merged**